app/controllers/katello/products_controller.rb

```diff
@@ -12,34 +12,6 @@ def section_id
       'contents'
     end
 
-    def available_repositories
-      if @product.custom?
-        render_bad_parameters _('Repository sets are not available for custom products.')
-      else
-        task = sync_task(::Actions::Katello::RepositorySet::ScanCdn, @product, params[:content_id])
-
-        repos = task.output[:results]
-        repos = exclude_rolling_kickstart_repos(repos)
-        repos = available_synced_repos(repos, params[:content_id])
-
-        locals = {:scan_cdn => task, :repos => repos, :error_message => nil}
-        locals[:error_message => task_error(task)] if task.result == 'warning'
-        render :partial => 'katello/providers/redhat/repos', :locals => locals
-      end
-    end
-
-    def toggle_repository
-      action_class = if params[:repo] == '1'
-                       ::Actions::Katello::RepositorySet::EnableRepository
-                     else
-                       ::Actions::Katello::RepositorySet::DisableRepository
-                     end
-      task = sync_task(action_class, @product, @content, substitutions)
-      render :json => { :task_id => task.id }
-    rescue => e
-      render :partial => 'katello/providers/redhat/enable_errors', :locals => { :error_message => e.message}, :status => 500
-    end
-
```

Comment:
I think this whole controller can be deleted

Reply:
Nice, thanks, you know how I love deleting stuff.

Reply:
@jlsherrill updated.